common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -139,6 +140,19 @@ public void setExternalLogger(final ILoggerCallback externalLogger) {
         mExternalLogger = externalLogger;
     }
 
+    /**
+     * Get only the required metadata from the DiagnosticContext
+     * to plug it in the log lines.
+     * Here we are considering the correlation_id and the thread_name.
+     * The need for this is because DiagnosticContext contains additional metadata which is not always required to be logged.
+     *
+     * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
+     */
+    public static String getDiagnosticContextMetadata() {
+        return DiagnosticContext.THREAD_NAME + " : " + DiagnosticContext.getRequestContext().getOrDefault(DiagnosticContext.THREAD_NAME, "UNSET") + ", " +
```

Comment:
Nit: wide lines; can we condense to <100 chars?

Reply:
will do!